src/context/ContextGetHelp.jsx

```diff
@@ -77,12 +99,7 @@ class ContextGetHelp extends React.Component {
             else if (this.state.selectedIndex > 0) {
                 e.preventDefault();
                 e.stopPropagation();
-
-                // the parent 'get help' option is not included in the getHelpOptions array
-                // but it is included as a selectedIndex, so there is an off by one that needs
-                // to be calculated, hence the -1
-                this.state.getHelpOptions[this.state.selectedIndex-1].onSelect();
-                return this.props.state;
+                return this.expand();
```

Comment:
Is there a reason we changed this?  The onSelect is supposed to be defined for all the getHelpOptions.  In this case, if there is ever more than one getHelpOption, which we plan on having, this will not work because it will always expand.  We should find a way to keep this reusable with a generic get help menu and one that is provided custom options.

Reply:
Fixed